types/flowjs/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for flowjs
+// Type definitions for flowjs 2.13.1
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
Makes sense. Thank you.

Reply:
I just changed the version because enabling lint rules results in lot of errors like interfaces starting for `I`. The version I made is for TS 3.1+ and has these fixed. Hopefully it's not a problem.